src/Symfony/Component/Security/Core/Authorization/AccessDecisionManagerInterface.php

```diff
@@ -37,6 +37,8 @@ public function decide(TokenInterface $token, array $attributes, $object = null)
      * @param string $attribute An attribute
      *
      * @return bool true if this decision manager supports the attribute, false otherwise
+     *
+     * @deprecated Since version 2.8, to be removed in 3.0.
```

Comment:
AFAIK we do not uppercase the first letter here, should be `@deprecated since version 2.8, to be removed in 3.0.`